packages/amplication-git-utils/src/types.ts

```diff
@@ -141,3 +135,19 @@ export interface CreateCommitArgs {
   branchName: string;
   files: UpdateFile[];
 }
+
+export interface OneBranchArgs extends FindOneRepositoryArgs {
```

Comment:
Why do we need to extend? What is the relation between OneBranchArgs and FindOneRepositoryArgs? I don't think there is any relation. Args of a function may change in one function without any change in the other, so this relation is not helping and just make things harder

Reply:
You cant find branch without the repository, so instead of rewrite the owner and the repo we just extends